rgd/geodata/tests/conftest.py

```diff
@@ -9,8 +9,13 @@
 
 
 @pytest.fixture
-def api_client() -> APIClient:
-    return APIClient()
+def api_client(user) -> APIClient:
+    client = APIClient()
+    user.is_superuser = True
+    user.is_staff = True
+    user.is_active = True
+    client.force_authenticate(user=user)
+    return client
```

Comment:
I could see the `authenticated_api_client` useful when testing permissions, but this is not being done right now. Perhaps it would be better to remove it entirely until then.

Reply:
Instead of changing `api_client`, maybe it'd be better to create a separate `admin_api_client` (or some similar name). I think it could be dangerous to write tests exclusively against an admin user, unless you expect all real users will have the same level of permissions.

Reply:
Resolved in 33dd776621e75bd8365c09753391d650b8cec5c8.